testing/python/metafunc.py

```diff
@@ -170,6 +170,13 @@ def test_idmaker_autoname(self):
         result = idmaker((py.builtin._totext("a"), "b"), [({}, b'\xc3\xb4')])
         assert result == ['a0-\\xc3\\xb4']
 
+    @pytest.mark.issue1437
```

Comment:
Thanks for trying to keep this style, but it's not really used anymore so this can be removed.

Instead of this, please right a short description in the docstring and mention #1437 there.